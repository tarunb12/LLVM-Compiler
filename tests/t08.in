// continue test

int main() {
    int i
    for (i = 0; i < 30; i = i + 1) {
        printf("i = %d -> i * 7 = %D\n", i, i * 7)
        if (i < 24) continue
        else break
        printf("if ur reading this continue or break did not work")
    }
    return 0
}