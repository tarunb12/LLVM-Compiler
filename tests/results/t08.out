i = 0 -> i * 7 = 0
i = 1 -> i * 7 = 7
i = 2 -> i * 7 = 14
i = 3 -> i * 7 = 21
i = 4 -> i * 7 = 28
i = 5 -> i * 7 = 35
i = 6 -> i * 7 = 42
i = 7 -> i * 7 = 49
i = 8 -> i * 7 = 56
i = 9 -> i * 7 = 63
i = 10 -> i * 7 = 70
i = 11 -> i * 7 = 77
i = 12 -> i * 7 = 84
i = 13 -> i * 7 = 91
i = 14 -> i * 7 = 98
i = 15 -> i * 7 = 105
i = 16 -> i * 7 = 112
i = 17 -> i * 7 = 119
i = 18 -> i * 7 = 126
i = 19 -> i * 7 = 133
i = 20 -> i * 7 = 140
i = 21 -> i * 7 = 147
i = 22 -> i * 7 = 154
i = 23 -> i * 7 = 161
i = 24 -> i * 7 = 168
